nucypher/policy/conditions/context.py

```diff
@@ -116,20 +118,31 @@ def get_context_value(context_variable: str, **context) -> Any:
     return value
 
 
-def _resolve_context_variable(param: Union[Any, List[Any]], **context):
+def resolve_any_context_variables(
+    param: Union[Any, List[Any], Dict[Any, Any]], **context
+):
     if isinstance(param, list):
-        return [_resolve_context_variable(item, **context) for item in param]
-    elif is_context_variable(param):
-        return get_context_value(context_variable=param, **context)
+        return [resolve_any_context_variables(item, **context) for item in param]
+    elif isinstance(param, dict):
+        result = {}
+        for k, v in param.items():
+            result[k] = resolve_any_context_variables(v, **context)
+        return result
```

Comment:
It's only stylistic but while we are code golfing here :-)

```suggestion
        return {k, resolve_any_context_variables(v, **context) for k, v in param.items()}
```